src/main/resources/js/condec.context.menu.js

```diff
@@ -0,0 +1,346 @@
+		document.getElementById("condec-context-menu-change-type-item").onclick = function() {
+			conDecDialog.showChangeTypeDialog(id);
+		};
+
+		document.getElementById("condec-context-menu-link-item").onclick = function() {
+			conDecDialog.showLinkDialog(id);
+		};
+
+		document.getElementById("condec-context-menu-delete-link-item").onclick = function() {
+			var parentId = findParentId(id);
+			conDecDialog.showDeleteLinkDialog(id, parentId);
+		};
+
+		document.getElementById("condec-context-menu-delete-item").onclick = function() {
+			conDecDialog.showDeleteDialog(id);
+		};
+
+		document.getElementById("condec-context-menu-set-root-item").onclick = function() {
+			if (window.conDecIssueModule !== undefined) {
+				window.conDecIssueModule.setAsRootElement(id);
+			} else if (window.conDecKnowledgePage !== undefined) {
+				window.conDecKnowledgePage.setAsRootElement(id);
+			}
+		};
+
+		document.getElementById("condec-context-menu-open-jira-issue-item").onclick = function() {
+			if (window.conDecKnowledgePage !== undefined) {
+				window.conDecKnowledgePage.openJiraIssue(id);
+			}
+		};
+	}
+	
+	ConDecContextMenu.prototype.createContextMenu = createContextMenu;
+	
+	function createContextMenuForSentences(posX, posY, id) {
+		isContextMenuOpen = true;
+		console.log("contextmenu opened");
+
+		$("#condec-context-menu-sentence").css({
+			left : posX,
+			top : posY
+		});
+		document.getElementById("condec-context-menu-sentence").style.zIndex = 9998;
+		document.querySelector("#condec-context-menu-sentence").setAttribute('aria-hidden', 'false');
+
+		document.getElementById("condec-context-menu-sentence-edit-item").onclick = function() {
+			// TODO
+		};
+
+		document.getElementById("condec-context-menu-sentence-delete-link-item").onclick = function() {
+			// TODO
+		};
+
+		document.getElementById("condec-context-menu-sentence-convert-item").onclick = function() {
+			conDecAPI.createIssueFromSentence(id, conDecObservable.notify);
+		};
+
+		document.getElementById("condec-context-menu-sentence-irrelevant-item").onclick = function() {
+			// TODO
+		};
+	}
+
+	ConDecContextMenu.prototype.createContextMenuForSentences = createContextMenuForSentences;
+
+	// export ConDecContext
+	global.conDecContextMenu = new ConDecContextMenu();
+})(window);
+
+function getSelectedTreeViewerNode(position) {
+	console.log("view.context.menu.js getSelectedTreeViewerNode");
+	var selector = position.reference.prevObject.selector;
+	return jQueryConDec("#jstree").jstree(true).get_node(selector);
+}
+
+function getSelectedTreeViewerNodeId(node) {
+	console.log("view.context.menu.js getSelectedTreeViewerNodeId");
+	console.log(getSelectedTreeViewerNode(node).data.id);
+	return getSelectedTreeViewerNode(node).data.id;
+}
+
+function getSelectedTreantNodeId(options) {
+	console.log("view.context.menu.js getSelectedTreantNodeId");
+	var context = options.$trigger.context;
+	console.log(context.id);
+	return context.id;
+}
+
+var changeKnowledgeTypeAction = {
```

Comment:
@Jcl15 , I think you can delete this function for now because the user can also use the edit element function.